pom.xml

```diff
@@ -142,56 +146,18 @@
                 <version>4.1.0.alpha1</version>
             </dependency>
             <dependency>
-                <groupId>com.vaadin.framework8</groupId>
+                <groupId>com.vaadin.framework7</groupId>
```

Comment:
Yes. another subproject of the same root. That is just a copy from fw8 spring demo.